app/routes/__index/certificate/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { Heading } from '@chakra-ui/react';
+
+export default function CertificateIndexRoute() {
+  return (
+    <div>
```

Comment:
To all files that uses `div` we should replace them to use `Box` if we're following conventions

Reply:
I don't think there is any point to changing them in the new pages since they will be replaced when they are updated based on the figma designs